package.json

```diff
@@ -219,6 +219,12 @@
           "markdownDescription": "Allows XML schemas/ DTD to be associated to file name patterns. Please refer to [XML file association with XSD](command:xml.open.docs?%5B%7B%22page%22%3A%22Validation%22%2C%22section%22%3A%22xml-file-association-with-xsd%22%7D%5D) or [XML file association with DTD](command:xml.open.docs?%5B%7B%22page%22%3A%22Validation%22%2C%22section%22%3A%22xml-file-association-with-dtd%22%7D%5D) for more information. \n\nExample:\n```json\n[{\n  \"pattern\": \"file1.xml\",\n  \"systemId\": \"path/to/file.xsd\"\n},\n{\n  \"pattern\": \"**/*.xsd\",\n  \"systemId\": \"http://www.w3.org/2001/XMLSchema.xsd\"\n}]\n```",
           "scope": "window"
         },
+        "xml.foldings.includeClosingTagInFold": {
+          "type": "boolean",
+          "default": true,
```

Comment:
I think the default value should be false because in vscode, the fold of last line provides this issue https://github.com/redhat-developer/vscode-xml/issues/690 In Eclipse IDE it will use the default value becaise it works properly https://github.com/microsoft/vscode/issues/3352#issuecomment-1085543997